config/collector.yaml

```diff
@@ -119,6 +132,15 @@ processors:
       - set(attributes["top_level_controller_type"], "CronJob") where resource.attributes["k8s.cronjob.name"] != nil
       - set(attributes["top_level_controller_name"], resource.attributes["k8s.cronjob.name"]) where resource.attributes["k8s.cronjob.name"] != nil
 
+# When sending telemetry to the GCP OTLP endpoint, the gcp.project_id resource attribute is required to be set to your project ID.
+  resource/gcp_project_id:
+    attributes:
+    - key: gcp.project_id
+      # MAKE SURE YOU REPLACE THIS WITH YOUR PROJECT ID
```

Comment:
I don't think these comments are necessary.  We use env var substitution to set this.